bootstrap.py

```diff
@@ -104,9 +107,41 @@ def main(coderoot: str, argv: Sequence[str] | None = None) -> int:
                 ),
                 exit=True,
             )
-        if not CI:
-            # TODO: full end-to-end testing once bootstrap is complete
-            print("Installing sentry's brew dependencies...")
-            proc.run_stream_output(("brew", "bundle"), cwd=f"{coderoot}/sentry")
 
+        print("Installing sentry's brew dependencies...")
+        proc.run_stream_output(("brew", "bundle"), cwd=f"{coderoot}/sentry")
+
+        # TODO: start colima silently in background here so that CI can be faster
+        # proc.fire_and_forget
+
+        # this'll create the virtualenv if it doesn't exist
+        proc.run_stream_output(("devenv", "sync"), cwd=f"{coderoot}/sentry")
+
+        # make bootstrap should be ported over to devenv sync,
+        # as it applies new migrations as well and so would need to ensure
+        # the appropriate devservices are running
+        proc.run_stream_output(
+            (
+                shell,
+                "-i",
+                "-euo",
+                "pipefail",
```

Comment:
we shouldn't need to run a shell  `venv/bin/activate` just sets `PATH=venv/bin:$PATH`